packages/plugin-eslint/src/lib/eslint-plugin.spec.ts

```diff
@@ -1,7 +1,51 @@
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+import type { SpyInstance } from 'vitest';
 import { eslintPlugin } from './eslint-plugin';
 
 describe('eslintPlugin', () => {
```

Comment:
Non-existent `eslintrc` path behaviour could also be tested.

Reply:
Test case added.